handlers/update_resource.go

```diff
@@ -66,18 +87,48 @@ func (d *updateResourceEntry) mergeJSON(maps ...*datautils.JSONObject) datautils
 				} else {
 					result[k] = v
 				}
-			case json.Number:
-				// Cast "version" to int, otherswise dynamodbattribute.MarshalMap will cast it to String
-				// See https://github.com/aws/aws-sdk-go-v2/issues/115
-				i64, err := v.(json.Number).Int64()
-				if err != nil {
-					panic(err)
-				}
-				result[k] = int(i64)
 			default:
 				result[k] = v
 			}
 		}
 	}
 	return result
 }
+
+func (d *updateResourceEntry) buildNewResourceVersion(newResource *datautils.Resource, existingResource *datautils.Resource) {
+	newUUID, err := identifier.NewUUIDService().Mint()
```

Comment:
Can we call this variable `tacoIdentifier` to distinguish it from `identifier.sdrUUID`?